consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/converter/XmlToObjectConverter.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.moj.sdt.cmc.consumers.converter;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import java.io.IOException;
+import org.springframework.stereotype.Component;
+
+@Component
+public class XmlToObjectConverter {
+
+    public <T> T  convertXmlToObject(String xml,
+                                     Class<T> targetClass) throws IOException {
+        XmlMapper xmlMapper = new XmlMapper();
+        return xmlMapper.readValue(xml, targetClass);
+    }
+
+    public String convertXmlToJson(String xml) throws IOException {
+        XmlMapper xmlMapper = new XmlMapper();
+        ObjectMapper jsonMapper = new ObjectMapper();
```

Comment:
Is it worth creating fields on the class for the XmlMapper and ObjectMapper rather than re-declaring them each time?

Reply:
done